package.json

```diff
@@ -54,13 +54,21 @@
     "prettier": "2.2.1",
     "prettier-plugin-solidity": "1.0.0-beta.10",
     "pretty-quick": "^2.0.1",
-    "secp256k1": "^3.7.1",
+    "secp256k1": "^3.8.1",
     "solc-0.8": "npm:solc@^0.8.4",
     "typescript": "^4.9.5",
     "web3": "^1.6.0"
   },
   "resolutions": {
-    "sse4_crc32": "npm:@node-rs/crc32@1.0.0"
+    "sse4_crc32": "npm:@node-rs/crc32@1.0.0",
+    "**/ganache/secp256k1": "4.0.4",
+    "**/eccrypto/secp256k1": "3.8.1",
+    "**/eth-crypto/secp256k1": "5.0.1"
+  },
+  "overrides": {
```

Comment:
do `resolutions` and `overrides` allow us to overcome the dependencies from uma/common?

Reply:
it removes the secp256k1 issue but it only allows us to redirect deps